### PR TITLE
To-Do List with Javascript

### PR DESCRIPTION
**Pull request by sergioobolevich**
In this branch, I have created the basement for a complete To-Do List application.
Projects and Tasks can be added, as well as due dates for tasks and priority (low, medium, or high).
You can open a task by clicking on its name to see description details and modify it.

I have used 2 modules: one for tasks and one for projects.
LocalStorage was implemented to persist changes.

### CODE REVIEW

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️ 🛑 
Hello, @chubaquelo very nice work. I love the design. You have done a good job so far. Now let's make it even better. Kindly check the following improvements to be made ⬇️ 

- [x]  Kindly remove commented code. In your toDoModule.js there is some commented code. This will keep your project clean.
- [x]  Kindly remove the node_modules in your project. I can see you have added it to the gitignore. But still, the node_modules were pushed to your repo. Kindly fix this.

lock_files_requirement (1)

- [x] Kindly separate your dom-manipulation from your logic. I can see you have two separate modules. However, both have logic and dom-manipulation. Kindly separate the two

modules_requirement (1)

- [x] I can see your project uses webpack 👏🏿 . However, you have not configured the scripts well in your package.json. Kindly check below ⬇️ 
`
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

`
Just like you see above, your package.json should be configured in such away.

### [OPTIONAL]

- [x] Kindly add validations to your forms. At the moment a user can create empty projects and to do's.


### if you have any question, kindly reach me on slack @peterrobert

### HAPPY CODING 💻

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      </div>
+    </header>
+
+    <div class="new-task-txt-input-div">
+      <input
+        placeholder="Add your new task here..."
+        type="text"
+        id="input-todo-name"
+      />
+      <select id="select-project"></select>
```
Comment (christianotieno) [inline dist/index.html]: ## Remove unnecessary fields
- [x] Kindly remove unnecessary busy elements on the page to make it less busy. 
- [x] The `add new task` below the project is unnecessary as the functionality has been implemented in the form up above. Kindly remove it.
- [x] The `show` and `order by` fields in the top need also to be removed for the same reason.
Screenshot 2021-02-06 115510

File: dist/style.css [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+.project-card button:hover {
+  background-color: greenyellow;
+  color: #000;
+}
+
+/* .project-card button:focus {
+  background-color: greenyellow;
+  color: #000;
+} */
+
```
Comment (christianotieno) [inline dist/style.css]: ## Remove Commented out code
- [x] Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @chubaquelo 👋 

Kindly take a look at some of the issues I have highlighted below based on my review. 
They are based on some elements in the page that ought to be removed as they are confusing.

### Happy coding 💻 
#### Christian Otieno (CR) :octocat:

**Review by SpaYco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**